core/src/main/resources/windows-service/jenkins-slave.xml

```diff
@@ -23,15 +23,22 @@ THE SOFTWARE.
 -->
 
 <!--
-  Windows service definition for Jenkins slave
+  Windows service definition for Jenkins agent.
+  The actual version of this definition has been moved to https://github.com/jenkinsci/windows-slave-installer-module/blob/master/src/main/resources/org/jenkinsci/modules/windows_slave_installer/jenkins-slave.xml.
```

Comment:
I do not see any usages, but this file can be downloaded as a resource.
So I'd prefer to keep it up to date